targettest/conio.c

```diff
@@ -9,6 +9,10 @@
  *
  */
 
+#if defined(__APPLE2__) && !defined(__APPLE2ENH__)
+/* Have the grid characters adapt to the availability of MouseText. */
```

Comment:
I propose a comment along these lines:
_Box drawing characters are usually constant expressions. However, there are scenarios where this is not the case. To ensure compatibility with code that assumes they are constant expressions, the scenarios in question must be explicitly enabled by defining DYN_BOX_DRAW. Currently, the only such scenario is the apple2 target. There, DYN_BOX_DRAW can be used to enable the use of MouseText characters on exactly those machines that support them._